### PR TITLE
Update docs based on new features & copy

### PR DESCRIPTION
**Pull request by colfax23**

### CODE REVIEW

File: docs/index.mdx [hunk 1]
```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
 
 <HowSvg />
 
+### Building on GitPOAP
+
+On-chain identities are only useful if there are others to interact with and applications built to leverage them.  That’s why we decided to build on top of POAP.
```
Comment (peebeejay) [inline docs/index.mdx]: On-chain identities are only useful if there are others people to interact with and applications built to leverage them. That’s why we've decided to build on top of POAP.

File: docs/index.mdx [hunk 2]
```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
 
 <HowSvg />
 
+### Building on GitPOAP
+
+On-chain identities are only useful if there are others to interact with and applications built to leverage them.  That’s why we decided to build on top of POAP.  
+
+GitPOAP is meant to be a platform that will enable a new era of web3 applications, and we’re excited to see what else people build on top of it.  Our public API is [now live](https://docs.gitpoap.io/api)!  Start building and feel free to reach out if there’s anything we can do to better support you.
```
Comment (peebeejay) [inline docs/index.mdx]: GitPOAP is a platform that will enable a new era of web3 applications, and we’re excited to see what people build on top of it. Our public API is [now live](https://docs.gitpoap.io/api)! Start building and feel free to reach out if there’s anything we can do to better support you.

File: docs/index.mdx [hunk 3]
```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
...
+
+### Why is this important?
+
+We’re building GitPOAP to solve real-world challenges we’ve faced while bootstrapping and growing open source projects.  web3 provides new modes of interaction and we believe GitPOAP can enable a novel way to reward contributors based on their value creation.  We believe this model can be generalized beyond open source and also enable new paradigms of collaboration.
```
Comment (peebeejay) [inline docs/index.mdx]: capital W for web3? 

`Web3 provides new modes of interaction and GitPOAP enables a novel way to reward contributors based on the value they add.`

File: docs/index.mdx [hunk 4]
```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
...
+
+The future of web3 requires deeper and more nuanced personas & reputation.  The on-chain representation of *you* is currently very limited, which puts a ceiling on how we can interact with one another & what applications we can build. 
+
+For example, many DAOs are relying on token holdings for their governance.  Imagine if you could easily poll the people who are actually contributing to your project?  Project funding is also inherently difficult - wouldn’t it be nice to have a unified & verified record of who has contributed, so you can channel the funds effectively?
```
Comment (peebeejay) [inline docs/index.mdx]: `...who's contributed...`

File: docs/index.mdx [hunk 5]
```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
...
+
+Finding collaborators is another major challenge. People are multidimensional, and understanding whether someone will be a good collaborator requires going much deeper than a resume.  We’re laying the foundation for a trusted, queryable record of contribution can enable new and novel ways to find others to work with.
```
Comment (peebeejay) [inline docs/index.mdx]: `We’re laying the foundation for a trusted, queryable record of contribution *that* can enable new and novel ways for you to find others to work with.`

File: docs/index.mdx (hunk 3 above)
Comment (colfax23) [reply docs/index.mdx]: I've seen web3 and Web 2.0 spelled each that way in many places, so I think i'm going to leave it if that's cool?

Comment (colfax23) [reply docs/index.mdx]: I made the other change tho 💪

**Review by peebeejay — APPROVED** [review]

**Pull request merged**